packages/kit/src/exports/index.js

```diff
@@ -72,3 +73,23 @@ export function text(body, init) {
 export function fail(status, data) {
 	return new ActionFailure(status, data);
 }
+
+/**
+ * Populate a route ID with params to produce relative path.
+ * @example
+ * ```js
+ * resolvePath(
+ *   `/blog/[slug]/[...somethingElse]`,
+ *   {
+ *     slug: 'hello-world',
+ *     somethingElse: 'something/else'
+ *   }
+ * ); // `/blog/hello-world/something/else`
+ * ```
+ * @param {string} id
+ * @param {Record<string, string>} params
+ * @returns {string}
+ */
+export function resolvePath(id, params) {
+	return resolve_entry(id, params);
```

Comment:
rather than calling the function indirectly like this, we should either a) re-export it or b) put the implementation here, and use that internally instead of `resolve_entry` (probably better, though a more significant change)

Reply:
Agreed, let's put the implementation here. Makes it more clear that changing the implementation affects the public API.